samples-generator/lib/createBatchTableHierarchy.ts

```diff
@@ -142,16 +160,65 @@ function createBatchTableHierarchy(options) {
             }
         }
         var batchedMesh = Mesh.batch(clonedMeshes);
+
+        if (use3dTilesNext || useGlb) {
```

Comment:
Is `use3dTilesNext` sufficient?

```suggestion
        if (use3dTilesNext) {
```